src/components/interaction-designer/BuilderCanvas.vue

```diff
@@ -31,45 +31,13 @@ const MARGIN_HEIGHT_CORRECTION_PX = -10
 const MARGIN_WIDTH_CORRECTION_PX = 120
 
 const DEBOUNCE_SCROLL_TIMER_MS = 300
-const DEBOUNCE_VALIDATION_TIMER_MS = 300
+// const DEBOUNCE_VALIDATION_TIMER_MS = 300
 
 @Component({})
 export class BuilderCanvas extends Vue {
   @Prop() block!: IBlock
   @Prop({default: 0}) widthAdjustment!: number
 
-  // ###### Validation API Watchers [
```

Comment:
Moved these into InteractionDesigner.vue so any changes under that component will call the validation, meaning changes in BuilderCanvas.vue and in ResourceViewer.vue